esriscanner.py

```diff
@@ -351,7 +353,7 @@ def add_to_report(message):
     print(message, file=report_target)
 
 log = None
-produce_report = False
+produce_report = True
```

Comment:
The reason this is set to False initially is that (as I understood it) you only want the report or email in cases where you encounter an error - the idea is that any error reported gets paired with setting this var.

Reply:
Oh, oops, I misunderstood.  I thought this had to be set to True in order to trigger reporting.